bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -76,6 +76,8 @@ void PrePrepareMsg::calculateDigestOfRequests(Digest& digest) const {
     for (const auto& sod : sigOrDigestOfRequest) {
       sigOrDig.append(sod.first, sod.second);
     }
+    LOG_INFO(GL, "Block timestamp: " << b()->time << "sigOrDig");
```

Comment:
I suggest to keep this at the DEBUG level, otherwise it may spam the logs when the system is under load.

Reply:
Removed